src/SFA.DAS.AssessorService.Application.UnitTests/Handlers/Certificates/StartCertificateHandlerTests/StartCertificateHandlerTests.cs

```diff
@@ -55,6 +55,8 @@ public void SetUp()
             // Arrange
             request.StandardUId = null;
             learnerRecord.FundingModel = 81;
+            learnerRecord.GivenNames = CapitaliseFirstLetterOfName(learnerRecord.GivenNames);
```

Comment:
Now that you have _mockCertificateNameCapitalisationService you can mock the ProperCase to just return it's input which would be more correct since you do not care what it does either way

Reply:
I have not done this because the algorithm should not run _at all_ in these tests. Setting it up the return the input value conflicts with the fact that it shouldn't run at all.